examples/tests/copy.earth

```diff
@@ -0,0 +1,167 @@
+
+FROM alpine:3.11
+WORKDIR /test
+
+all:
+    BUILD +copy1
+    BUILD +copy2
+    BUILD +copy3
+    BUILD +copy4
+    BUILD +copy5
+    BUILD +copy6
+    BUILD +copy-art1
+    BUILD +copy-art2
+    BUILD +copy-art3
+    BUILD +copy-art4
+    BUILD +copy-art5
+    BUILD +copy-art6
```

Comment:
I mean its fine; but can we name them after what they are testing?

Reply:
Done